nav2_amcl/include/nav2_amcl/motion_model/omni_motion_model.hpp

```diff
@@ -19,8 +19,8 @@
  *
  */
 
-#ifndef DIFFERENTIAL_MOTION_MODEL_HPP
-#define DIFFERENTIAL_MOTION_MODEL_HPP
+#ifndef NAV2_AMCL__MOTION_MODEL__OMNI_MOTION_MODEL_HPP_
+#define NAV2_AMCL__MOTION_MODEL__OMNI_MOTION_MODEL_HPP_
```

Comment:
I just changed this from diff_motion_model to omni_motion_model, I hope this should be fine.